src/jdk.compiler/share/classes/com/sun/tools/javac/resources/compiler.properties

```diff
@@ -1917,6 +1917,11 @@ compiler.err.is.preview=\
 compiler.warn.is.preview.reflective=\
     {0} is a reflective preview API and may be removed in a future release.
 
+# 0: symbol, 1: symbol
+compiler.warn.restricted.method=\
```

Comment:
I've updated the name of this key (from `compiler.warn.restricted.method.call` to make it more general, and also applicable in the case of method reference. Note that the warning text itself has not changed, as that was already correct.